packit_service/worker/helpers/testing_farm.py

```diff
@@ -557,6 +565,28 @@ def run_testing_farm(
 
         return TaskResults(success=True, details={})
 
+    def retry_on_submit_failure(self, message: str) -> TaskResults:
+        """
+        Retry when there was a failure when submitting TF tests.
+
+        Args:
+            message: message to report to the user
+        """
+        interval = (
+            BASE_RETRY_INTERVAL_IN_MINUTES_FOR_OUTAGES * 2**self.celery_task.retries
+        )
+
+        self.report_status_to_tests(
+            state=BaseCommitStatus.pending,
+            description=message + " The task will be"
+            f" retried in {interval} {'minute' if interval == 1 else 'minutes'}.",
+        )
```

Comment:
Isn't this too long for a description? I am worried this will not be visible. What about using markdown content for that purpose?

Reply:
good point, I moved the reason for failure to the `markdown_content` to be safe